runner/run_benchmarks.py

```diff
@@ -4,12 +4,27 @@
 import statistics
 import subprocess
 import sys
+from importlib.metadata import PackageNotFoundError, version
 from pathlib import Path
 
 import requests
 import yaml
 
 
+def get_solver_version(solver_name):
+    try:
+        if solver_name == "highs":
+            return version("highspy")
+        elif solver_name == "scip":
+            return version("PySCIPOpt")
+        elif solver_name == "gurobi":
+            return version("gurobipy")
+        else:
+            return version("glpk")
+    except PackageNotFoundError:
+        return ValueError(f"Package Not Found Error {solver_name}")
```

Comment:
```suggestion
        elif solver_name == "glpk":
            return version("glpk")
        else:
            raise NotImplementedError(f"We do not yet support {solver_name}")
```

Again, it's better to explicitly raise an error like in my suggestion so that when we add a new solver we get a meaningful error. Also note that you `raise` an error, not `return` it.